model/articles.model.js

```diff
@@ -10,3 +10,35 @@ exports.fetchArticleById = (article_id) => {
       return rows[0];
     });
 };
+
+exports.fetchArticles = (sort_by = "created_at", order = "desc") => {
+  const validSortByValues = [
+    "article_id",
+    "title",
+    "topic",
+    "author",
+    "created_at",
+    "votes",
+    "comment_count",
+  ];
+
+  let sqlQuery = `
+  SELECT articles.article_id, articles.title, articles.topic, articles.author, 
```

Comment:
If the column name is unique between the two tables, you don't need the `table.` before the column name.